GLE_solver.py

```
@@ -1,167 +1,305 @@
+        w_bc=args.w,
+        Delta=args.delta,
+        ngrid=args.ngrid,
+        tolerance=args.tolerance
+    )
+    
+    if solution is not None:
+        print("\nSolution found successfully!")
+        
+        # Analyze solution
+        x0, theta_min, _ = find_x0_and_theta_min(solution.x, solution.y[1, :])
+        print(f"\nSolution properties:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)